FrameworkSystem/scripts/dirac-proxy-init.py

```diff
@@ -117,23 +116,23 @@ def addVOMSExtIfNeeded( self ):
     if not vomsAttr:
       return S_ERROR( "Requested adding a VOMS extension but no VOMS attribute defined for group %s" % self.__piParams.diracGroup )
 
-    result = VOMS.VOMS().setVOMSAttributes( self.__proxyGenerated, attribute = vomsAttr, vo = Registry.getVOMSVOForGroup( self.__piParams.diracGroup ) )
-    if not result[ 'OK' ]:
-      return S_ERROR( "Could not add VOMS extensions to the proxy\nFailed adding VOMS attribute: %s" % result[ 'Message' ] )
+    vomsAttributes = VOMS.VOMS().setVOMSAttributes( self.__proxyGenerated, attribute = vomsAttr, vo = Registry.getVOMSVOForGroup( self.__piParams.diracGroup ) )
+    if not vomsAttributes[ 'OK' ]:
+      return S_ERROR( "Could not add VOMS extensions to the proxy\nFailed adding VOMS attribute: %s" % vomsAttributes[ 'Message' ] )
 
     gLogger.notice( "Added VOMS attribute %s" % vomsAttr )
-    chain = result['Value']
+    chain = vomsAttributes['Value']
     chain.dumpAllToFile( self.__proxyGenerated )
     return S_OK()
 
   def createProxy( self ):
     gLogger.notice( "Generating proxy..." )
-    result = ProxyGeneration.generateProxy( piParams )
-    if not result[ 'OK' ]:
-      gLogger.error( result[ 'Message' ] )
+    proxyGenerated = ProxyGeneration.generateProxy( piParams )
+    if not proxyGenerated[ 'OK' ]:
+      gLogger.error( proxyGenerated[ 'Message' ] )
       sys.exit( 1 )
-    self.__proxyGenerated = result[ 'Value' ]
-    return result
+    self.__proxyGenerated = proxyGenerated[ 'Value' ]
```

Comment:
Brrrr... was much more clear before. I think that it is important for standard things to keep standard patterns. This should be a strong convention.